File: app/src/main/java/com/tydeya/familycircle/data/familyInteractor/abstraction/FamilyNetworkInteractorCallback.java
package com.tydeya.familycircle.data.familyInteractor.abstraction;

import com.tydeya.familycircle.domain.familymember.FamilyMember;

import java.util.ArrayList;

public interface FamilyNetworkInteractorCallback {

    void memberDataFromServerUpdate(ArrayList<FamilyMember> members);
}

File: app/src/main/java/com/tydeya/familycircle/domain/familyinteraction/FamilyNetworkInteractorCallback.kt
package com.tydeya.familycircle.domain.familyinteraction

import com.tydeya.familycircle.data.family.Family
import com.tydeya.familycircle.data.familymember.FamilyMember
import com.tydeya.familycircle.utils.Resource

interface FamilyNetworkInteractorCallback {

    fun familyDataUpdated(familyServerResource: Resource<Family>)

    fun familyMembersUpdated(membersServerResource: Resource<ArrayList<FamilyMember>>)
}